From 8c4364c853c9a21fd97843090e8b6219528f6b8b Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Mon, 24 Apr 2023 11:15:13 +0800
Subject: [PATCH] Replace Vulkan enum value, fix packaging issue

---
 render/vulkan/util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/render/vulkan/util.c b/render/vulkan/util.c
index 8c803d8..5d002c9 100644
--- a/render/vulkan/util.c
+++ b/render/vulkan/util.c
@@ -45,7 +45,7 @@ const char *vulkan_strerror(VkResult err) {
 	ERR_STR(ERROR_INVALID_EXTERNAL_HANDLE);
 	ERR_STR(ERROR_FRAGMENTATION);
 	ERR_STR(ERROR_INVALID_OPAQUE_CAPTURE_ADDRESS);
-	ERR_STR(PIPELINE_COMPILE_REQUIRED);
+	ERR_STR(PIPELINE_COMPILE_REQUIRED_EXT);
 	ERR_STR(ERROR_SURFACE_LOST_KHR);
 	ERR_STR(ERROR_NATIVE_WINDOW_IN_USE_KHR);
 	ERR_STR(SUBOPTIMAL_KHR);
-- 
2.33.1

